### ISSUE TITLE
allow players to "overwrite" the beacon menu

### ISSUE THREAD

**Issue by bmorel** [Unvanquished/Unvanquished#2451]
Since there is now a way to give bots instructions, it would be nice for them to be controlled more easily by players, and the beacon menu seems to be the most appropriate, currently existing, interface for this.
OTOH not all players will want to drive bots this way.

So I think it would be interesting if players could "attach" custom commands to it's entries.

**Comment by cu-kai** [comment]
Maybe simply give an option (checkbox which sets a cvar) in the beacon menu to "Control bot players with these beacons" or something? If checked, any use of `/beacon` would locally trigger some sort of /tactic use too.

Here are my thoughts on the matter, in any case:
 - "Pointer" simply has the same functionality of @sweet235's `stay-here` tactic, and bots will attempt to defend that location.
 - "Timer" could simply be a timed pointer - bots will `stay-here` at that location after the countdown finishes.
 - "Defend" could have the bots defend a specific structure (little difference to pointer, really) or follow/defend a specific player.
 - "Repair" could have the bots grab construction kits to repair the structure labelled by the beacon.
 - "Attack" is a difficult one. It would ideally to act like a pointer, or instead have a disclaimer that telling bots to "attack" is simply a command to get the bots to constantly rush the enemy base.

We'd need many more changes to `/tactic` and the bot behaviour tree including the inclusion of @sweet235's `stay-here` and `follow` tactics in upstream to make this remotely achievable.

**Comment by sweet235** [comment]
I considered using the beacons for bot commands, but didn't use them yet since

- It is unclear how to specify how many bots should be affected
- Some people might not know that they are giving bot commands

I will make a PR for `stay-here` and `follow` soon™.

**Comment by bmorel** [comment]
> It is unclear how to specify how many bots should be affected

True, one needs to be able to not control all the team at once. Maybe with a kind of of slider-control (that can also be moved with the wheel, to go faster)?

> Some people might not know that they are giving bot commands

IMO the main problem to solve would be that people communicate very little, and beacons becoming useful for bots could possibly lead to people using them without bots, too.

Ofc, I think this needs an alternate beacons menu to answer. Unv needs something to say "yes", "no", "sorry", "no problem", "go away" and that kind of stuff without using the slow team-chat.
This is probably a different topic though.
For information, in red-eclipse where there are more beacons menus, they *are* used (there are 2 "general" beacon menus with 8 entries each, and 1 "team" beacon which have different number of entries depending on the game mode).
This is probably also tied to the fact beacon menus there are **much more faster** to use.
